test/manual-test-examples/webgl/loadSTL/loadASCII/sketch.js

```diff
@@ -0,0 +1,19 @@
+let teapot;
```

Comment:
I think that the ASCII and binary manual tests can be combined for ease of viewing. Also, I'd move the models to the assets folder one directory up for consistency.